src/content/api/batch-types/farmlands-external-asset.mdx

```diff
@@ -85,6 +85,10 @@ Exported Farmlands Credit Card information used for importing and updating of a
     Unique 9 digit field to display in barcode format to pay by [Patron Code](/api/patron-codes).
   </Property>
 
+  <Property name="number" type="bignumber">
+    Unique 16 digit field used in conjunction with `expiry` to pay by [Patron Code](/api/patron-codes).
```

Comment:
What do you think about 'Unique 16 digit identifier for the card that can be used for eCommerce payments'? pay by patron code makes me think of quick pay or the scan barcode payment flow and this feels separate even though implementation wise patron code might be the correct way it is modeled.

Reply:
I like it. I'm going to keep in the mention that it must be used in conjunction with Expiry, I think that's important info and isn't related to quick pay.